src/command.ts

```diff
@@ -0,0 +1,88 @@
+import * as cmd from "./cmd.ts";
+import { addPrComment } from "./github.ts";
+
+const validCommandRegex = /^@GiteaBot run `([a-zA-Z0-9]+)`$/;
```

Comment:
I think we should restrict it to `make` commands.
Otherwise, this sounds like the mother of all security issues for whoever runs the bot.
Or how are we currently running the bot?

Reply:
Don't we trust maintainers?

Reply:
The idea is to restrict it to collaborators.

Reply:
But generally, just `make` commands would work for me as well. 90% of cases it will be `make fmt` or similar.